spec/factories/facilities.rb

```diff
@@ -32,5 +32,20 @@
       zone { Faker::Address.block }
       facility_type { "PHC" }
     end
+
+    transient do
```

Comment:
Tests don't know how to create regions for `states` and `blocks`, since they are only created through the API endpoints. Create these regions by default.